src/realm/realmReducer.js

```diff
@@ -11,6 +11,9 @@ import {
 } from '../actionConstants';
 import { objectFromEntries } from '../jsBackport';
 
+// Value copied from zerver/models.py in 2021-10.
+const DEFAULT_MESSAGE_CONTENT_EDIT_LIMIT_SECONDS = 600;
```

Comment:
I hesitated to pick 0 because it's not a valid value in our representation in the Redux state, which is modeled on what servers 5.0+ send, and I thought that might be confusing or cause problems.

Reply:
Maybe 1?

Reply:
Sure.